.gitignore

```diff
@@ -13,3 +13,4 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+/.idea/*
```

Comment:
you probably also want to ignore this project's binary file

Reply:
Will be anything else besides .exe?

Reply:
hah I forgot that .exe is a thing :D I would add `tsm` and `tsm.exe` then ;)